library/src/main/java/net/yslibrary/licenseadapter/LicenseEntry.java

```diff
@@ -2,15 +2,9 @@
 
 import android.os.Parcelable;
 
-/**
- * Created by yshrsmz on 2016/04/18.
- */
 public interface LicenseEntry extends Parcelable {
-
   void load();
 
-  boolean isLoaded();
```

Comment:
it's not acceptable for v1.4 since this is a public api change.
so just marking as deprecated is better.